src/check_release.rs

```diff
@@ -396,6 +373,41 @@ pub(super) fn run_check_release(
                 Color::Ansi(AnsiColor::Red),
                 true,
             )?;
+        } else {
```

Comment:
If errors or warnings got printed, let's make sure we leave an empty line between the last of those and the `Summary` line to improve readability. Otherwise the summary looks visually attached to the most recently printed error or warning block.